install_shapepipe

```diff
@@ -200,26 +202,28 @@ check_conda() {
     echo -ne "${RED}ERROR: Found Conda version $CONDA_VERSION but require 4.0.0 or greater.${NC}\n"
     exit 1
   fi
-  # Check Conda minor version
-  if [ "$CONDA_VERSION_MINOR" -gt 4 ]
+  # Look for conda.sh file
+  if [ -f "$CONDA_PREFIX_1$CONDA_SH" ]
```

Comment:
$CONDA_PREFIX_1 does not seem to exist elsewhere in this script.

Reply:
fixed!